src/components/GroupModal.js

```diff
@@ -201,6 +201,11 @@ class GroupModal extends Component {
       return contextChildren;
     }
 
+    const accumulator = (total, card) => {if(card.details.price){return total + card.details.price} return total}
```

Comment:
This is kind of a nitpick but this would be cleaner as `total + (card.details.price || 0)`

Reply:
Fair point, made the change.